ChakraCore.Debugger.nuspec

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>ChakraCore.Debugger</id>
+    <version>$version$</version>
+    <description>Debugging companion library for the ChakraCore JavaScript engine.</description>
+    <authors>Microsoft</authors>
+    <projectUrl>https://github.com/Microsoft/ChakraCore-Debugger</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+  </metadata>
+
+  <files>
+    <file src="lib\Debugger.Protocol\**\*.h" target="include\Chakra\Debugger\Protocol" />
+    <file src="lib\Debugger.ProtocolHandler\**\*.h" target="include\Chakra\Debugger\ProtocolHandler" />
+    <file src="lib\Debugger.Service\**\*.h" target="include\Chakra\Debugger\Service" />
```

Comment:
Any reason to put `\Chakra` in the path?  That feels potentially redundant.

Reply:
No objective reason.
I figured headers could be included as, say, `#include <Chakra/Debugger/ChakraDebugService.h>`.

Reply:
Ahh, I see.  That's why I made the two public headers explicitly include `ChakraDebug` so that the pathing wouldn't be strictly necessary.  I don't feel too strongly one way or the other though.